_posts/2016-10-06-virtual-mesh-hack-night.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Virtual Mesh Hack Night
+slug: virtual-mesh-hack-night
+text: Join our virtual mesh from anywhere and play with distributed services.
+location: "#virtualmesh:tomesh.net"
+locationMap: https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net
```

Comment:
Removed `locationMap` but kept the `location`, otherwise the pin icon doesn't go away and it looks weird.